vlcplayer/__init__.py

```diff
@@ -40,14 +40,23 @@ def __init__(self):
     def playytb(self, vid):
         self.play(vid2youtubeMRL(vid))
 
-    def play(self, mrl):
-        media = self.instance.media_new(mrl)
-        media_list = self.instance.media_list_new([mrl])
-        self.player.set_media(media)
+    def play(self, mrl_string):
+        mrl = mrl_string.split(";")
+        #media = self.instance.media_new(mrl)
+        media_list = self.instance.media_list_new(mrl)
+        #self.player.set_media(media)
```

Comment:
Please remove these two commented line, the are not useful.

Reply:
updated